src/numutils.c

```diff
@@ -3,16 +3,42 @@
 #include "../include/numutils.h"
 #include "../include/arithmetic.h"
 
+bstd_number* bstd_number_from_int(int value, uint64_t length, bool isSigned) {
+
+    bstd_number* number = malloc(sizeof(bstd_number));
+
+    (*number) = (bstd_number) {
+        .value = 0,
+        .scale = 0,
+        .length = length,
+        .isSigned = isSigned,
+        .positive = false
+    };
+
+    bstd_assign_int(number, value);
+
+    return number;
+}
+
 int bstd_number_is_integer(const bstd_number* number) {
     return !number->scale;
 }
 
-int64_t bstd_number_to_int(const bstd_number* number) {
+static int64_t number_to_signed_value(const bstd_number* number) {
```

Comment:
agree with the static. We need to make sure it is also done done for other helper functions.